### DISCUSSION TITLE
tooltips

### DISCUSSION THREAD

**Discussion by pbwn** [zim-desktop-wiki/zim-desktop-wiki#1669] [General]
anyone using ubuntu wants to disable tooltips in zim: 
add the following to ~/.config/gtk-3.0/gtk.css
tooltip {
    opacity: 0;
}
it worked for me

**Comment by jaap-karssenberg** [comment]
They are there because a longer page name could not fit the side pane
width, so tooltip informs you in case name is truncated. They cannot be
turned off.

**Reply by pbwn** [reply]
zim is wonderful!

**Comment by jaap-karssenberg** [comment]
Well, I find them useful and I also happen to be the person writing most of
the app in my spare time and providing it for free. You're totally free to
use another application or submit a patch for zim if you feel this is such
a burden to you.

Kind regards,

Jaap

**Comment by pbwn** [comment]
zim is wonderful!

**Comment by jaap-karssenberg** [comment]
Donations are appreciated and they go to good causes but my work on this
project is taking my spare time at no pay. And donations are voluntary. So
yes: free.

If you make your feedback constructive usually people willing to tweak
something for you. However ridiculing choices is not the best way to get
people to do you a favor.


Op do 7 okt. 2021 17:29 schreef p m brown ***@***.***>:

**Reply by pbwn** [reply]
youre the best Jaap!

**Comment by jaap-karssenberg** [comment]
Fixed in  36151156203fa150d45bf88c79af554cf457ac69, will end up in the next release.

You're welcome :)
